### PR TITLE
Prepare 3.1.0-dev.0 release with compatibility to Cate Web API 3.1

### PR DESCRIPTION
**Pull request by forman**

### CODE REVIEW

File: src/renderer/components/DataSourceItem.tsx [hunk 1]
```diff
@@ -51,8 +51,8 @@ const DataSourceItem: React.FC<DataSourceItemProps> = ({dataSource, showDataSour
     }
 
     let typeSpec;
-    if (dataSource.typeSpecifier) {
-        typeSpec = <span style={TYPE_SPEC_STYLE}>{dataSource.typeSpecifier}</span>;
+    if (dataSource.dataType) {
+        typeSpec = <span style={TYPE_SPEC_STYLE}>{dataSource.dataType}</span>;
```
Comment (TonioF) [inline src/renderer/components/DataSourceItem.tsx:55]: Can you change TYPE_SPEC_STYLE to DATA_TYPE_STYLE, please?

**Review by TonioF — CHANGES_REQUESTED** [review]
Just one change request, otherwise good!

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+### Changes 3.1.0
```
Comment (AliceBalfanz) [inline CHANGES.md:1]:
```suggestion
### Changes 3.1.0 (in development)
```

**Review by AliceBalfanz — COMMENTED** [review]
Minor change request :) Otherwise approve

**Pull request merged**